go/vt/mysqlctl/schema.go

```diff
@@ -41,15 +41,24 @@ import (
 
 var autoIncr = regexp.MustCompile(` AUTO_INCREMENT=\d+`)
 
-// executeSchemaCommands executes some SQL commands, using the mysql
-// command line tool. It uses the dba connection parameters, with credentials.
-func (mysqld *Mysqld) executeSchemaCommands(sql string) error {
-	params, err := mysqld.dbcfgs.DbaConnector().MysqlParams()
+// executeSchemaCommands executes some SQL commands. It uses the dba connection parameters, with credentials.
+func (mysqld *Mysqld) executeSchemaCommands(ctx context.Context, sql string) error {
+	conn, err := getPoolReconnect(ctx, mysqld.dbaPool)
 	if err != nil {
 		return err
 	}
-
-	return mysqld.executeMysqlScript(params, strings.NewReader(sql))
+	defer conn.Recycle()
+	_, more, err := conn.ExecuteFetchMulti(sql, 0, false)
```

Comment:
@rohit-nayak-ps @mattlord It was something 🤦 here. The problem is that `executeSchemaCommands` gets multiple queries separated with `;` as input and we didn't properly read off all results which means they are not properly processed immediately and we get the weird behavior we saw.

I changed this to call `ExecuteFetchMulti` and with draining all results and things are now properly passing and there's no weird behavior anymore with missing tables.